test/BrainPass.t.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.13;
+
+import {PRBTest} from "prb-test/PRBTest.sol";
+import {Cheats} from "forge-std/Cheats.sol";
+import {stdError} from "forge-std/Errors.sol";
+import {BrainPassCollectibles} from "../src/BrainPass/BrainPass.sol";
+import {ERC721TokenReceiver} from "solmate/tokens/ERC721.sol";
+import {MockERC20} from "../lib/solmate/src/test/utils/mocks/MockERC20.sol";
+
+contract TestEditor is PRBTest, Cheats {
```

Comment:
change the name.

test suite its still weak. a basic smell its that you have around 10 custom errors in the contract and i can see only 4 selectors here. So at least there are 6 cases that you are not validating.